OpenRA.Mods.Common/Traits/Buildable.cs

```diff
@@ -34,6 +34,9 @@ public class BuildableInfo : TraitInfo<Buildable>
 		[Desc("Force a specific faction variant, overriding the faction of the producing actor.")]
 		public readonly string ForceFaction = null;
 
+		[Desc("Sequence of the actor that contains the icon.")]
+		[SequenceReference] public readonly string Icon = "icon";
```

Comment:
`IconSequence` maybe.
This doesn't have to be changed because you're just moving code around, obviously.